localisation/000_wc_military_localization.csv

```diff
@@ -343,6 +343,7 @@ wc_ret_cul_bloodtroll;War Mothers;;Kriegsm
 we_ret_cul_tuskarr;Turtle Cossacks;;Schildkr�tenkosaken;;;;;;;;;;;x
 wc_ret_cul_vrykul;Thanes;;Thanen;;;;;;;;;;;x
 wc_ret_cul_vulpera;Merchant Gunner;;Handelssch�tzen;;;;;;;;;;;x
+wc_ret_cul_frost_giant;Champions of Hodir;;;;;;;;;;;;;x
```

Comment:
What if frost giants are not titanic?
For example, if they worship the old gods?

Reply:
changed to more neutral